etc/profile-a-l/firefox-common-addons.profile

```diff
@@ -74,7 +74,6 @@ whitelist ${HOME}/.zotero
 whitelist ${HOME}/dwhelper
 whitelist /usr/share/lua
 whitelist /usr/share/lua*
-whitelist /usr/share/vulkan
```

Comment:
@kmk3

IMO we should keep `whitelist /usr/share/vulkan` here. The include logic is a bit more complicated than usual. Only `firefox.profile` includes `wusc`, most of the other profiles for Firefox-based web browsers do NOT:

- abrowser
- basilisk
- cliqz
- cyberfox
- firedragon
- icecat
- palemoon
- waterfox

We could add `wusc` in the above profiles to fix things, but we'd have to test it on all of these web browsers to make sure. Not easily done (time-wise).

Reply:
No worries, that can happen. It caught my eye because I'm working on a branch to streamline/fix/unclutter this include logic in our Firefox-based web browser profiles. It's a bit of a mess IMO currently and mistakes like this can easily happen to anyone. Thanks for fixing!